presto-main/src/main/java/io/prestosql/execution/QueryStateTimer.java

```diff
@@ -36,6 +42,9 @@
     private final AtomicReference<Long> beginResourceWaitingNanos = new AtomicReference<>();
     private final AtomicReference<Long> beginDispatchingNanos = new AtomicReference<>();
     private final AtomicReference<Long> beginPlanningNanos = new AtomicReference<>();
+    private final AtomicReference<Long> beginPlanningCpuNanos = new AtomicReference<>();
+    private final AtomicReference<Long> beginPlanningThreadId = new AtomicReference<>();
+    private final AtomicReference<Long> beginStartingThreadId = new AtomicReference<>();
```

Comment:
you don't need to store `starting thread id`. Just `beginPlanningThreadId` is enough